packages/gatsby/src/commands/develop.ts

```diff
@@ -102,14 +102,14 @@ class ControllableScript {
 let isRestarting
 
 module.exports = async (program: IProgram): Promise<void> => {
-  const developProcessPath = require.resolve(`./develop-process`)
+  const developProcessPath = slash(require.resolve(`./develop-process`))
   // Run the actual develop server on a random port, and the proxy on the program port
   // which users will access
   const proxyPort = program.port
   const developPort = await getRandomPort()
 
   const developProcess = new ControllableScript(`
-    const cmd = require("${developProcessPath}");
+    const cmd = require(developProcessPath);
```

Comment:
```suggestion
    const cmd = require("${developProcessPath}")
```
need to revert this - this is inside template string literal